docs/02-app/02-api-reference/02-file-conventions/01-metadata/robots.mdx

```diff
@@ -59,7 +59,7 @@ Sitemap: https://acme.com/sitemap.xml
 ## Customize user-agent rules
```

Comment:
I think instead of having a separate title, you can combine as as a 2nd example under `## Generates a Robots File` since it's a part of generating a robots file.

Reply:
Thanks for reviewing and suggesting changes.
I have added another commit which removes the title markup for customize user-agent rules and kept it as an another example ; as suggested .

Reply:
Thanks for adding this @808vita 🙏🏼 

I think it should be fine to add a nested heading inside the `Generate a Robots File`. Since we show how to customize all with `'*'` in the first example, we can then show how to customize specific user agents with the array. 

Some suggestions below:

Reply:
Thanks @delbaoliveira for suggesting changes .
I have committed your proposed changes.

Reply:
Thank you!